### ISSUE TITLE
Encrypted messages in unfriendly formats for external webmail services

### ISSUE THREAD

**Issue by StuartLRM** [lavab/web#882]
Description
---------------
When sending an encrypted message to an external mail service (i.e. not lavaboom) the message is headers and body are broken into two encrypted attachments. When decoded these messages include HTML tags and are not friendly to read. I would say they are not zero knowledge :-/

Reproduction Steps
--------------------------
1. Create an email 
In Lavaboom add a new contact including their public key
Send that contact an email

**Comment by martinchodev** [comment]
Same problem here when sending to Gmail.
I receive only the manifest.pgp file and in the body of the message I get "This is an encrypted email, open it here if your email client doesn't support PGP manifests:" and then a link that sends me to a page that says "Sorry, you've encountered a missing feature! ".

I'm using GnuPG in my computer (Linux) and if I decode the manifest file I get a JSON file that contains the sender and destination emails and a hash of the message.

**Comment by andreis** [comment]
PGP manifests are what we use to encrypt email metadata (i.e. headers and mime info), but currently there are still some compatibility issues. There is [a discussion](http://lists.gnupg.org/pipermail/gnupg-devel/2015-June/030036.html) on `gnupg-devel` about standardising this and making it backwards-compatible. Once that discussion settles we'll implement _Memory Hole_ and hopefully this issue will be fixed.

**Comment by StuartLRM** [comment]
Hi Andreis - I had a quick look at the discussion - it looks like it's primarily concerned with headers - does it also address the issue raised above of style markup being encoded in the message? E.g. application-pgp-encrypted-attachment containing stuff like:

````<section class="email"><section class="body">This is the encrypted body message</section><footer class="signature"><br class=""><br class="">````

Apols if it does and I've missed it

**Comment by Nexus6** [comment]
Is there any news on this?  The people who I've convinced to use PGP mail use either T-bird/Enigmail or Mailvelope, and both fail in the described way when processing received encrypted emails from Lavaboom.  This makes it impossible to use the commercial Lavaboom service with them as there's no chance that I'm going to get them to migrate to using the service as their main email system.

**Comment by andreis** [comment]
@Nexus6 Yeah, we're going through a back-end refactoring, and in the process we'll drop the current encoding scheme in favour of PGP/MIME - until [Memory Hole](https://lists.gnupg.org/pipermail/gnupg-devel/2015-June/030036.html) is finalized.
